### PR TITLE
Add recipe for neo4jsbml 0.10.1

### PR DESCRIPTION
**Pull request by guillaume-gricourt**
Please consider neo4jsbml
A tool created to import SBML files into Neo4j
Thanks

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/neo4jsbml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  commands:
+    - {{ name }} --help
+
+about:
+  home: https://github.com/brsynth/{{ name }}
+  summary: Import SBML file into Neo4j
+  license: MIT
+  license_file: LICENSE
+  doc_url: https://{{ name }}.readthedocs.io/
+  dev_url: https://github.com/brsynth/{{ name }}
```
Comment (dopplershift) [inline recipes/neo4jsbml/meta.yaml:45]: Can you remove the templating here? It makes it very hard to copy/paste the URLs for testing.

File: recipes/neo4jsbml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install -vv .
+  entry_points:
+    - "neo4jsbml=neo4jsbml.__main__:main"
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - python >=3.8
```
Comment (dopplershift) [inline recipes/neo4jsbml/meta.yaml:23]: Duplicate:
```suggestion
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple minor tweaks and this is good to go.

I also need confirmation from @tduigou that they are willing to be listed as a maintainer for this recipe.

File: recipes/neo4jsbml/meta.yaml (hunk 1 above)
Comment (guillaume-gricourt) [reply recipes/neo4jsbml/meta.yaml]: fix 6cf4f5b1c6

**Review by dopplershift — APPROVED** [review]

**Pull request merged**